lib/NetPacks.h

```diff
@@ -1738,6 +1738,11 @@ struct ObstaclesRemoved : public CPackForClient
 
 	template <typename Handler> void serialize(Handler &h, const int version)
 	{
+		if(version < 777 && !h.saving)

```

Comment:
We do not use backward compatibility for NetPacks, server and client must have same version.